launch/state_machines/demo_siza.launch

```diff
@@ -42,7 +42,7 @@
 
     <node name="conversation_engine" pkg="conversation_engine" type="engine" output="log" required="true" args="--robot amigo">
         <param name="robot" value="hero" />
-        <param name="challenge" value="siza_demo" />
+        <param name="challenge" value="challenge_demo" />
```

Comment:
What will remain the difference between demo_intern/free_mode/demo_siza after this commit?

Reply:
Essentially demo_siza (needs a name change) is a demo_intern with a fully loaded worldmodel.

Reply:
This is for the knowledge. Is there a knowledge file of siza demo?